src/util/metrics/threads_linux.rs

```diff
@@ -382,8 +264,17 @@ mod tests {
 
         tids.iter()
             .find(|t| {
-                Stat::collect(pid, **t)
-                    .map(|stat| stat.name == name)
+                pid::stat_task(pid, **t)
+                    .map(|stat| stat.command == name)
+                    .unwrap_or(false)
+            })
+            .unwrap();
+
+        tids.iter()
+            .find(|t| {
+                pid::io_task(pid, **t)
```

Comment:
You can use `TempDir` to write and read some files.

Reply:
I tried to read `Cargo.lock` lol